app/src/main/java/org/mozilla/focus/home/pocket/PocketVideoFragment.kt

```diff
@@ -4,21 +4,57 @@
 
 package org.mozilla.focus.home.pocket
 
+import android.graphics.Color
 import android.os.Bundle
 import android.support.v4.app.Fragment
+import android.support.v7.widget.RecyclerView
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import kotlinx.android.synthetic.main.fragment_pocket_video.view.*
 import org.mozilla.focus.R
 
 /** A feed of Pocket videos. */
 class PocketVideoFragment : Fragment() {
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
-        return inflater.inflate(R.layout.fragment_pocket_video, container, false)
+        val layout = inflater.inflate(R.layout.fragment_pocket_video, container, false)
+        layout.videoFeed.gridView.adapter = PocketVideoAdapter()
+        return layout
     }
 
     companion object {
         fun create() = PocketVideoFragment()
     }
 }
+
+private class PocketVideo(
+        val title: String,
+        val url: String,
+        val imageUrl: String
```

Comment:
maybe this should contain the videoSource too (possibly with a default)? But yeah, I understand it's just an early version.

Reply:
This will get finalized when we merge the data, #767.

Reply:
Not sure if you saw but in #771, Kenny mentioned giving us a `tv_url` - will be rolled out next week 🎉 and updated in the API doc.